specs/electra/beacon-chain.md

```diff
@@ -1111,9 +1136,9 @@ def process_execution_payload(state: BeaconState, body: BeaconBlockBody, executi
     assert execution_engine.verify_and_notify_new_payload(
         NewPayloadRequest(
             execution_payload=payload,
-            execution_requests=body.execution_requests,  # [New in Electra]
             versioned_hashes=versioned_hashes,
             parent_beacon_block_root=state.latest_block_header.parent_root,
+            execution_requests_hash=compute_execution_requests_hash(body.execution_requests),  # [New in Electra]
```

Comment:
I'd consider moving the call of `compute_execution_requests_hash` to `verify_and_notify_new_payload` but its not critical